kiosk/README.md

```diff
@@ -0,0 +1,54 @@
+
+|Key|Purpose|
+|---------------------------------|-|
+|GameDataUrl                      | Path to the game data JSON file with details about the games to include.|
+|PlayUrlRoot                      | Left part of the embedded player URL up to and including --run.|
+|HighScoresToKeep                 | The maximum number of high scores to keep for a given game.|
+|HighScoreInitialsLength          | The number of initials to allow users to enter.|
+|HighScoreInitialAllowedCharacters| The list and order of characters to allow in high scores.|
+|GamepadPollLoopMilli             | How often the gamepads are polled for user interaction with the kiosk menus and other inputs. If this is too large, actions may be missed, such as a quick button press. If it's too small, then a single action may be interpretted as multiple (like a single button press counting twice).|
+|GamepadCacheMilli                | How often the gamepad state is cached. Since multiple components access the browser's Navigator.getGamepads() API, we use a caching mechanism to optimize how often they're accessed. This value should be smaller than the gamepad polling loop.|
+|GamepadAButtonPin                | The pin index for the A button. |
+|GamepadBButtonPin                | The pin index for the B button.|
+|GamepadEscapeButtonPin           | The pin index for the Escape button. This button leaves a game without waiting and returns the user to the menu.|
+|GamepadResetButtonPin            | The pin index for the Reset button.|
+|GamepadMenuButtonPin             | The pin index for the Menu button. Note that since this menu is used by games, it does not return the user to the kiosk menu. |
+|GamepadLeftRightAxis             | The gamepad axis index for left/right detection.|
+|GamepadLeftRightThreshold        | The threshold to detect for a "right" action. It's negated to detect "left" actions.|
+|GamepadUpDownAxis                | The gamepad axis index for up/down detection.|
+|GamepadUpDownThreshold           | The threshold to detect for a "down" action. It's negated to detect "up" actions.|
+|Keyboard`Input`Keys              | Each is an array of string arrays matching the browser keys that map to the target behavior. The index of a list maps to the same gamepad index (player 1 is the first array at index 0, etc.) |
+
+## Localhost testing
+
+To test Kiosk locally:
+
+1. Ensure your pxt repo is up to date and has been built recently.
+2. In a command shell, in the `pxt` repo, cd into the `kiosk` folder and start the Kiosk dev server: `npm run start`. This will *not* open a browser window.
+3. In another command shell, in the `pxt-arcade` repo, start the Arcade dev server: `pxt serve --rebundle`. This will open the Arcade webapp in a browser.
+
+Requests to the `/kiosk` endpoint will be routed to the Kiosk dev server.
+
+Debug and step through Kiosk code using the browser dev tools (F12 to open).
+
+
+## Test in staging environment
```

Comment:
Might be fine to just swap these sections out with a link to https://github.com/microsoft/pxt-backend/blob/master/node/packages/backend/README.md#getting-your-access-token and just list the extra 'and now append --kiosk' here? Just to centralize instructions a little as these get out of sync easy / only apply to internal devs with access to backend anyways

Reply:
Makes sense to not dup this info. Will clean up in next iteration.